Use a real account for the demo experience

marcellofuschi opened the pull request:
Fixes #474
Puppeteer tests are yet to be changed

marcellofuschi committed:
Use a real account for the demo experience

adeebshihadeh reviewed (commented):
Nice red diff! Works pretty well. Should be good to merge once the tests are fixed up.

marcellofuschi committed:
Update puppeteer tests

marcellofuschi commented:
> Nice red diff! Works pretty well. Should be good to merge once the tests are fixed up.

Thanks, I updated the tests.

I deleted the `offline` one as it seems you aren't able to open the demo dashboard offline anymore.
